src/tasks/WasmAppBuilder/WasmAppBuilder.csproj

```diff
@@ -20,8 +20,12 @@
 
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" />
-    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="4.7.1" />
-    <PackageReference Include="System.Text.Json" Version="6.0.0" PrivateAssets="all" />
+    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="$(SystemReflectionMetadataLoadContextVersion)" />
```

Comment:
I realized we need to do the same for the MetadataLoadContext.  Can this reference a shared versions.props property of should if follow the suggested source-build conditioned pattern?